README.md

```diff
@@ -1,21 +1,56 @@
 # Macfly
 
-**TODO: Add description**
+This library handles Fly.io Macaroon encoding, decoding, attentuation, etc..
+
+[Hex Docs](https://hexdocs.pm/macfly/)
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `macfly` to your list of dependencies in `mix.exs`:
+The package can be installed by adding `macfly` to your list of dependencies in `mix.exs`:
 
 ```elixir
 def deps do
   [
-    {:macfly, "~> 0.1.0"}
+    {:macfly, "~> 0.2.15"}
   ]
 end
 ```
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at <https://hexdocs.pm/macfly>.
+## Usage
+
+### Encoding and Decoding
+
+```elixir
+# for example tokens, see the tests or test/vectors.json file
+token = "FlyV1 fm2_lJPE..."
+
+{:ok, [%Macfly.Macaroon{}] = macaroons} = Macfly.decode(token)
```

Comment:
I think this is meant for decoding a bundle of comma separated tokens. To decode a single token, you'd use `Macfly.Macaroon.decode/2`

Reply:
I think either one works, but I'll add a comment explaining both versions.

Reply:
I think we should steer people towards `Macfly.decode/1`. I'm just saying that it will often return more than one token, so the pattern wouldn't match here.